bundles/org.openhab.persistence.mongodb/src/test/java/org/openhab/persistence/mongodb/internal/DataCreationHelper.java

```diff
@@ -234,7 +257,7 @@ public static Stream<Arguments> provideOpenhabItemTypes() {
      * @return A stream of Arguments, each containing a DatabaseTestContainer instance.
      */
     public static Stream<Arguments> provideDatabaseBackends() {
-        if (DockerClientFactory.instance().isDockerAvailable()) {
+        if (DockerClientFactory.instance().isDockerAvailable() && isAVXSupported()) {
```

Comment:
Thanks for having a look! Based on the warning it may be possible to still run the tests using the mongodb 3.6 and 4.4 containers when AVX support is missing?

Reply:
@wborn I wouldn't bet on this, since you don't have control over the testcontainer, which could lead to a broken pipe later on.
It does run the tests on the mongo in memory backend, which does a great job with the standard ops.